tests/core/utils/test_extract_zip.py

```diff
@@ -17,7 +17,7 @@
 import py7zr
 import pytest
 
-from antarest.core.utils.utils import BadArchiveContent, extract_zip
+from antarest.core.utils.utils import BadArchiveContent, extract_archive
 
 
 class TestExtractZip:
```

Comment:
We must add unit tests for 7zip too

Reply:
It is already tested for `.7z`, I need to rename the class